includes/Admin/AnyCommentGenericSettings.php

```diff
@@ -425,74 +425,74 @@ public function init_settings () {
+				          ->set_description( esc_html( __( 'Let your users login to WordPress using your configured social networks.', "anycomment" ) ) ),
 
 
 				     $this->field_builder()
 				          ->checkbox()
 				          ->set_id( self::OPTION_SHOW_ADMIN_BAR )
-				          ->set_title( __( 'Show Admin Bar', "anycomment" ) )
-				          ->set_description( esc_html( __( 'Show admin bar for regular WordPress users and those who logged in via social.', "anycomment" ) ) ),
+				          ->set_title( __( 'Use Admin Bar', "anycomment" ) )
+				          ->set_description( esc_html( __( 'Show the admin bar to logged-in users, including those who logged in with a social profile.', "anycomment" ) ) ),
 
 
 				     $this->field_builder()
 				          ->checkbox()
 				          ->set_id( self::OPTION_SHOW_PROFILE_URL )
-				          ->set_title( __( 'Show Profile URL', "anycomment" ) )
-				          ->set_description( esc_html( __( 'Show link to user in the social media or website when available (name of the user will be clickable).', "anycomment" ) ) ),
+				          ->set_title( __( 'Use Profile URLs', "anycomment" ) )
+				          ->set_description( esc_html( __( 'Link commenter names to their WordPress or social media profile.', "anycomment" ) ) ),
 
 
 				     $this->field_builder()
 				          ->checkbox()
 				          ->set_id( self::OPTION_SHOW_TWITTER_EMBEDS )
 				          ->set_title( __( 'Display Twitter Embeds', "anycomment" ) )
-				          ->set_description( esc_html( __( 'Detect & display tweets from Twitter as embedded widget.', "anycomment" ) ) ),
+				          ->set_description( esc_html( __( 'Detect & display twitter URLS as comment embeds.', "anycomment" ) ) ),
 
 
 				     $this->field_builder()
 				          ->checkbox()
 				          ->set_id( self::OPTION_SHOW_VIDEO_ATTACHMENTS )
 				          ->set_title( __( 'Display Video Attachments', "anycomment" ) )
-				          ->set_description( esc_html( __( 'Display video link from comment as attachment.', "anycomment" ) ) ),
+				          ->set_description( esc_html( __( 'Display video links (YouTube etc) inline in the comments', "anycomment" ) ) ),
 
 
 				     $this->field_builder()
 				          ->checkbox()
 				          ->set_id( self::OPTION_SHOW_IMAGE_ATTACHMENTS )
 				          ->set_title( __( 'Display Image Attachments', "anycomment" ) )
-				          ->set_description( esc_html( __( 'Display image link from comment as attachment.', "anycomment" ) ) ),
+				          ->set_description( esc_html( __( 'Display attached images within the comments.', "anycomment" ) ) ),
 
 				     $this->field_builder()
 				          ->checkbox()
 				          ->set_id( self::OPTION_RATING_TOGGLE )
-				          ->set_title( __( 'Display Rating', "anycomment" ) )
-				          ->set_description( esc_html( __( 'Display 5 star rating above comments.', "anycomment" ) ) ),
+				          ->set_title( __( 'Feature top-rated comments', "anycomment" ) )
```

Comment:
We have titles capitalized first letter, maybe we could make it the same way here, so that we won't need to fix it for many languages later

Reply:
```suggestion
				          ->set_title( __( 'Feature Top-rated Comments', "anycomment" ) )
```